<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupBinary</class>
 <widget class="QMainWindow" name="PopupBinary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AuthMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(29, 29, 29)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="MessageLabel">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gadugi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: silver;</string>
      </property>
      <property name="text">
       <string>Message</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>246</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="AcceptButton">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: silver;
border-width: 2px;
border-style: solid;
border-color: rgb(52, 52, 52);
padding: 4px;
}

QPushButton:pressed{
padding: 2px;
}

</string>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RefuseButton">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: silver;
border-width: 2px;
border-style: solid;
border-color: rgb(52, 52, 52);
padding: 4px;
}

QPushButton:pressed{
padding: 2px;
}

</string>
        </property>
        <property name="text">
         <string>Refuse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
